utils/slow_documentation_tests.txt

```diff
@@ -1 +1,2 @@
+docs/source/en/generation_strategies.md
```

Comment:
The total time spent in this file is >120s. We may want to revisit it in the future (e.g. making all examples from the same model, so it only spends time loading one model)